drafts/use-cases/1.1.security-considerations.md

```diff
@@ -10,23 +10,30 @@ There are several possibilities here. One of them would be to use a *@graph* not
 
 ```json
 {
+    "@context": "/api/context.jsonld",
     "@graph": [
         {
             "@id": "/api",
-            "operation": [
-                {
-                    "@type": "hydra:Operation",
-                    "title": "List events",
-                    "method": "GET",
-                    "returns": "hydra:Collection"
-                },
-                {
-                    "@type": "hydra:Operation",
-                    "title": "Create new event",
-                    "method": "POST",
-                    "expects": "schema:Event"
-                }
-            ]
+            "@type": "hydra:EntryPoint",
+            "#events": {
```

Comment:
My line of thought is that anything not native to plain JSON will look alien to most people already familiar with plain JSON. Just the idea of using URIs as opaque identifiers (instead of UUIDs or integers) seem crazy to most people at first.

To keep the number of alien elements to a minimum, it would help to keep the syntax as plain as possible. I'm not decided on whether dropping the `@` prefix is a good idea or not, but it's worth investigating, imho.